test/extensions/filters/udp/dns_filter/dns_filter_fuzz_test.cc

```diff
@@ -0,0 +1,39 @@
+#include "common/common/logger.h"
```

Comment:
The corpus files above look empty. Is that intentional?

@asraa do you mind taking a quick look at this fuzz test?

Reply:
Oooh.  This is odd, I will look into this.  They absolutely should not be empty.   They are truncated in my repository as well.

Reply:
I'll update in the next push:
```
$ sha1sum dns_filter_corpus/*
016fac1e4a40199b26b08df73179f9249e6a680b  dns_filter_corpus/016fac1e4a40199b26b08df73179f9249e6a680b
110be4738f0cc29218ba95bd16a1442b57b3caaf  dns_filter_corpus/110be4738f0cc29218ba95bd16a1442b57b3caaf
497a3f29c3a53a65853a9e0ab3dd315fb92ac025  dns_filter_corpus/497a3f29c3a53a65853a9e0ab3dd315fb92ac025
fb9282f0af3341cfc98d56f10fffffd5529d8802  dns_filter_corpus/fb9282f0af3341cfc98d56f10fffffd5529d8802
```